pkg/daemon/multus/nginx-config.yaml

```
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: multus-validation-test-web-server-conf
+  labels:
+    app: multus-validation-test-web-server
+data:
+  server.conf: |
+    server {
+        listen       8080;
```

```suggestion
        listen       8080;
```
is 8080 is required, currently I have seen port 8080 conflicts with controller runtime matrix port.

---

controller runtime isn't running in the same pod as nginx, so there shouldn't be a conflict. Noted though, in case we do see an unexpected issue in the future.

And as a note, this is configured to use the commonly-used 8080 instead of 80 because security policies often prevent pods running with port 80.